src/extension.ts

```diff
@@ -329,6 +342,28 @@ export async function activate(
     },
   });
 
+  const provider = new DevToolsViewProvider(context.extensionUri);
+  context.subscriptions.push(
+    window.registerWebviewViewProvider(DevToolsViewProvider.viewType, provider),
+  );
+  let devtoolServer: Disposable | null = null;
```

Comment:
```suggestion
  let devtoolServers: Disposable | null = null;
```
[nit] Should we call this `devtoolsServer` since we're referring to it as plural elsewhere?

Reply:
This code disappears in #197 :)